src/rate.rs

```diff
@@ -573,8 +574,16 @@ impl QuantizerParameters {
     let scale = q57(QSCALE + bit_depth as i32 - 8);
     let quantizer = bexp64(log_target_q + scale);
     let (offset_u, offset_v) = chroma_offset(log_target_q);
-    let quantizer_u = bexp64(log_target_q + offset_u + scale);
-    let quantizer_v = bexp64(log_target_q + offset_v + scale);
+    let log_target_q_u = log_target_q + offset_u + scale;
+    let log_target_q_v = log_target_q + offset_v + scale;
+    let quantizer_u = bexp64(log_target_q_u);
+    let quantizer_v = bexp64(log_target_q_v);
```

Comment:
These values are subtly different from what we expect. For correct `lambda_u` and `lambda_v`, rather:
```rust
let log_target_q_u = log_target_q + offset_u;
let log_target_q_v = log_target_q + offset_v;
let quantizer_u = bexp64(log_target_q_u + scale);
let quantizer_v = bexp64(log_target_q_v + scale);
```